src/identity-providers/add/DescriptorSettings.tsx

```diff
@@ -248,7 +247,7 @@ const Fields = ({ readOnly }: DescriptorSettingsProps) => {
         ></Controller>
       </FormGroup>
 
-      {principalType.includes("ATTRIBUTE") && (
+      {principalType!.includes("ATTRIBUTE") && (
```

Comment:
Using a non-null assertion operator here causes the page to break if you add a new identity provider. Using optional chaining seems to fix the issue.
```suggestion
      {principalType?.includes("ATTRIBUTE") && (
```

Reply:
thanks... fixed